tests/function_attr_test.rs

```diff
@@ -0,0 +1,13 @@
+#[allow(dead_code)]
+#[test]
+fn tests_pass() {
+    let t = trybuild::TestCases::new();
+    t.pass("tests/function_component_attr/*-pass.rs");
```

Comment:
Pass tests should also only be run with MSRV.
Rustc may introduce additional lints over the time to fail pass tests.

Reply:
Tests breaking somehow to me sounds like a good idea, despite being slightly annoying upon trying to perform an unrelated PR ([already happened with clippy before](https://github.com/yewstack/implicit-clone/pull/35)). Maybe have them run under both MSRV and latest stable so we can see whether such a lint change occured?

Reply:
@kirillsemyonkin it's already what I did here

But in Yew we only run them against MSRV.

imo it's best to run it on both because we need the thing to keep working on stable and to be alerted asap when it's not. 

I don't mind restricting this test to MSRV though.

Reply:
@futursolo do you agree with the argument or you still prefer the passing test to run only on MSRV?